web/wp-content/themes/lfevents/library/lfe-functions.php

```diff
@@ -433,3 +433,106 @@ function lfe_fix_community_post( $post_id, $feed_id ) {
 	}
 }
 add_action( 'wprss_ftp_converter_inserted_post', 'lfe_fix_community_post', 10, 2 );
+
+/* Will only run on front end of site */
+if ( ! ( is_admin() ) ) {
+	/**
+	 * Make all JS defer onload (in conjunction with moving jQuery to footer).
+	 *
+	 * @param string $url the URL.
+	 */
+	function defer_parsing_of_js( $url ) {
+		if ( false === strpos( $url, '.js' ) ) {
+				return $url;
+		}
+
+			// this needs to match jquery file name, typically jquery.js.
+		if ( strpos( $url, 'jquery-3.4.1.min.js' ) ) {
+				return $url;
+		}
+
+			return "$url' defer onload='";
```

Comment:
can you explain how this works?

Reply:
1) Page looks broken
Looks like the function to move JS to the footer is being ignored as the GF widget is moving the jquery in to the widget. Generally Gravity forms + widgets messes with jQuery. The GF widget has an option to disable script output. 
For future consideration, widgets are pretty poorly implemented, users find them hard to find/use. I haven't used widgets in a site build for a couple of years. Often better to use an options panel, gutenberg, or ACF for better control. 
Background info - https://legacy.forums.gravityhelp.com/topic/gravity-form-widget-breaks-jquery // https://github.com/timber/timber/issues/1668

Fixed: I have removed the function to load scripts in footer.

2) Error Message

Fatal error: Cannot redeclare ub_getNamesOfPresentBlocks() (previously declared in /app/web/wp-content/plugins/ultimate-blocks/src/init.php:61) in /app/web/wp-content/plugins/ultimate-blocks/src/init.php on line 61

Caused by Ultimate Blocks. Not Stackable Ultimate Blocks. But Ultimate Blocks. plugins/ultimate-blocks
From the options page (hidden in WP Admin) you are only using this for the Countdown block. Not sure what's happening with the script, but it seems to be affected by my function that is moving JS scripts to the footer, which isn't right. The plugin seems to be incorrectly enqueuing the scripts, as it's also loading them on pages where the block isn't present (the error is present on many pages).

Fixed: I have removed the function to load scripts in footer. I have asked the plugin author for advice. I personally would not use this entire plugin just for the countdown (the countdown also has some responsive issues).

3) Title 
Background url wasn't being escaped properly (phpcs fix), but I changed the code and thats fixed.

4) Defer script
This added "defer" to all .js scripts loaded except for jquery. It will cause the browser to wait for the DOM tree to be complete before executing it, so helping speed up first paint speed.
Background info - https://blog.dareboost.com/en/2017/12/defer-scripts-to-speed-up-rendering/

Reply:
Author of the Ultimate Blocks plugin said "must be using old version, try to update to 2.1.2" - we are using Version 1.1.9.
This should resolve the error (although the GF Widget fix will still prevent us from using the render blocking JS move to footer function)

Reply:
BTW, I've upgraded Ultimate Blocks.